UnitySDK/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -281,7 +281,7 @@ public override void AgentReset()
         {
             bodyPart.Reset(bodyPart);
         }
-
+        GetRandomTargetPos();
```

Comment:
Why is this called at every step now.

Reply:
called on AgentReset because sometimes the target was falling off the platform